Dockerfile

```diff
@@ -0,0 +1,29 @@
+ARG RELEASE_TAG
+
+FROM ubuntu:latest AS downloader
+
+WORKDIR /download
+
+RUN apt-get update && apt-get install -y curl
+
+RUN curl -L -o ./llamafile-server https://github.com/Mozilla-Ocho/llamafile/releases/download/${RELEASE_TAG}/llamafile-server-${RELEASE_TAG}
+
+RUN chmod +x ./llamafile-server
+
+
+
+FROM scratch AS final
```

Comment:
`FROM scratch AS final`
does not work in this context. Made it work with :
`FROM ubuntu:latest AS final`

Reply:
thanks, I replaced `scratch` with `ubuntu:latest`